chromadb/api/__init__.py

```diff
@@ -56,17 +56,17 @@ def create_collection(
         self,
         name: str,
         metadata: Optional[CollectionMetadata] = None,
-        get_or_create: bool = False,
         embedding_function: Optional[EmbeddingFunction] = None,
+        get_or_create: bool = False,
```

Comment:
is it safe to just change the param order? Wouldn't that be a breaking API change?

Reply:
Oh this is aligning it with the impls - python overrides don't work - i see .